### PR TITLE
Add Chinese Translation

### PR DESCRIPTION
**Pull request by linjungz**
*Issue #, if available:*

*Description of changes:*
Add Chinese translation by putting contents in /contents/zh-cn

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/zh-cn/designs/command/_index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+IoT解决方案应该能以这样的方式与设备进行交互：解决方案或是使用该解决方案的人，可以可靠地要求设备执行某个动作。更进一步的，这样的交互需要在会出现间隙性中断的网络环境中进行，并通常需要使用资源非常有限的设备。
+
+## 解决方案
+
+IoT解决方案使用"命令"设计来要求设备执行操作，并通过利用一个简单的概念确保可靠的交互：除非被确认为成功，否则任何请求的操作都不会被视为成功。
+
+下图中显示的"命令"设计可以提供此功能：
+
+![Command Design](command.png) 
+([PPTx](atlas-command.pptx))
```
Comment (brettf) [inline content/zh-cn/designs/command/_index.md]: This is no longer how the designs should link to PPTx downloads. Instead you can just link to `https://github.com/aws/iot-atlas/blob/master/content/en-us/designs/iot-atlas-patterns.pptx` using `{{< ref path="/designs/iot-atlas-patterns.pptx" lang="en" >}}`. That single PPTx in the English language directory contains all the diagrams of the site's Designs.

File: content/zh-cn/designs/gateway/_index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## 挑战
+IoT解决方案中的[端点]({{< ref "/glossary/vocabulary#endpoint" >}})通常不具备直接连接互联网的能力，或者是运行在无法直接访问互联网的网络中。即使存在这些限制，从端点获取数据并与这些端点进行交互需要有相应的连通机制
+
+## 解决方案
+IoT解决方案使用"网关"设计来克服端点通常遇到的限制。通过这样的设计网关可以实现与端点的可靠及安全的通信
+
+如下图所示的"网关"设计可以提供这样的功能。图中服务器(Server)是运行在云中。网关(Gateway)则运行在设备可以访问的网络中
+
+![Gateway Design](gateway.png)
+([PPTx](atlas-gateway.pptx))
```
Comment (brettf) [inline content/zh-cn/designs/gateway/_index.md]: This is no longer how the designs should link to PPTx downloads. Instead you can just link to `https://github.com/aws/iot-atlas/blob/master/content/en-us/designs/iot-atlas-patterns.pptx` using `{{< ref path="/designs/iot-atlas-patterns.pptx" lang="en" >}}`. That single PPTx in the English language directory contains all the diagrams of the site's Designs.

File: content/zh-cn/designs/telemetry/_index.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+## 挑战
+IoT解决方案需要可靠和安全地接收不同设备在远程环境中测量的数据，并可能使用到不同的协议。另外，一旦接收到测量数据，解决方案就需要处理和路由这些数据以供解决方案的组件来使用。
+
+## 解决方案
+IoT解决方案使用"遥测"设计，通过支持成熟的通信协议来确保在间歇性中断的网络环境下能够传送测量数据，同时提供在不同测量频率和数据量的情况下可扩展的数据接收能力，以及将数据路由给其他组件使用。
+
+以下架构图展示的"遥测"设计可以提供这样的功能。
+
+![Telemetry Design](telemetry.png)
+([PPTx](atlas-telemetry.pptx))
```
Comment (brettf) [inline content/zh-cn/designs/telemetry/_index.md]: This is no longer how the designs should link to PPTx downloads. Instead you can just link to `https://github.com/aws/iot-atlas/blob/master/content/en-us/designs/iot-atlas-patterns.pptx` using `{{< ref path="/designs/iot-atlas-patterns.pptx" lang="en" >}}`. That single PPTx in the English language directory contains all the diagrams of the site's Designs.

File: content/zh-cn/designs/telemetry_archiving/_index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+## 挑战
+为了提供洞察力，IoT解决方案需要支持对其产生的信息进行实时、批处理和预测分析。在利用历史数据时，每种分析模式都能获得更好的信息。*同时*，未来还有目前未被理解的分析方法，因此IoT解决方案必须以尽可能灵活的方式存档数据，以满足未来的需求.
+
+## 解决方案
+IoT解决方案存储原始未处理过的传感器数据，并支持对这些原始样本进行有序的重放，从而确保业务可以获得最新且不断发展的洞察集合。存储和重放能力(store-and-replay)应使历史原始样本看起来几乎就像样本按正常时间序列到达一样。
+
+下图中显示的"遥测存档"设计可以提供此功能。
+
+![Telemetry Archiving Architecture](archiving.png) ([PPTx](atlas-archiving.pptx))
```
Comment (brettf) [inline content/zh-cn/designs/telemetry_archiving/_index.md]: This is no longer how the designs should link to PPTx downloads. Instead you can just link to `https://github.com/aws/iot-atlas/blob/master/content/en-us/designs/iot-atlas-patterns.pptx` using `{{< ref path="/designs/iot-atlas-patterns.pptx" lang="en" >}}`. That single PPTx in the English language directory contains all the diagrams of the site's Designs.

File: content/zh-cn/_footer.md [hunk 5]
```diff
@@ -0,0 +1 @@
+©2018, Amazon Web Services, Inc. 或其附属公司。保留所有权利。
```
Comment (brettf) [inline content/zh-cn/_footer.md]: This should be a 2019 copyright

File: content/zh-cn/designs/device_state_replica/_index.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+9. IoT解决方案中的组件可以从设备状态副本中请求到更新后且是当前的状态
+
+
+
+### 设备到组件的状态副本
+
+当*设备*是状态变更的发起者，而且状态变更需要通知到IoT解决方案中的组件时，以下设计可以为IoT解决方案所用：
+
+![Device-to-component State Replication](d2c-state.png)
```
Comment (brettf) [inline content/zh-cn/designs/device_state_replica/_index.md]: This should be `Device-to-component State Replica`

File: content/zh-cn/designs/gateway/_index.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+通常来说，网关上消息的本地存储和处理将遵循先进先出（即 **FIFO**）方法。话虽如此，但答案*可能*会有所不同，这具体取决于消息中实际包含的数据。在这种情况下，通过确定网关日志记录方法对实际报告数据的影响，可以帮助我们避免以后出现的问题。
+
+如下图所示，通常考虑三种类型的方法：**先进先出(FIFO)**, **剔除(Culling)** 和 **聚合(Aggregate)**。
+
+![Message Processing Algorithms](algorithms.png)
+
+**先进先出(FIFO)** – 这种[方法](https://en.wikipedia.org/wiki/FIFO_(computing_and_electronics))通常实现起来比较直接，在多种场景下都很有用。在这种方法的消息处理图中可以看到，数据从左侧到达，当本地存储空间满后，数据从右侧离开。具体数据的例子包括：维护测量数据和通用的遥测数据
+
+**剔除(Culling)** - 这种方法对于在丢失曲线细节时保留绝对点值非常有用。在消息处理图中，此方法的数据从左侧到达。一旦本地存储中已存放超过*剔除点*的数据，则一些清除逻辑会移除出其他所有样本（或每`N`个样本）。具体数据的例子包括: [千瓦(kW)](https://en.wikipedia.org/wiki/Watt#Kilowatt), [安培(Amperage)](https://en.wikipedia.org/wiki/Amperage), [伏特(Voltage)](https://en.wikipedia.org/wiki/Voltage), etc
```
Comment (brettf) [inline content/zh-cn/designs/gateway/_index.md]: Please ensure that these Wikipedia URLs are pointing to the Simplified Chinese version on Wikipedia (if that version exists).

File: content/zh-cn/designs/gateway/_index.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+#### 应该使用什么方法来存储后续需要发送的消息？ 
+
+通常来说，网关上消息的本地存储和处理将遵循先进先出（即 **FIFO**）方法。话虽如此，但答案*可能*会有所不同，这具体取决于消息中实际包含的数据。在这种情况下，通过确定网关日志记录方法对实际报告数据的影响，可以帮助我们避免以后出现的问题。
+
+如下图所示，通常考虑三种类型的方法：**先进先出(FIFO)**, **剔除(Culling)** 和 **聚合(Aggregate)**。
+
+![Message Processing Algorithms](algorithms.png)
+
+**先进先出(FIFO)** – 这种[方法](https://en.wikipedia.org/wiki/FIFO_(computing_and_electronics))通常实现起来比较直接，在多种场景下都很有用。在这种方法的消息处理图中可以看到，数据从左侧到达，当本地存储空间满后，数据从右侧离开。具体数据的例子包括：维护测量数据和通用的遥测数据
```
Comment (brettf) [inline content/zh-cn/designs/gateway/_index.md]: Please ensure that these Wikipedia URLs are pointing to the Simplified Chinese version on Wikipedia (if that version exists).

File: content/zh-cn/designs/gateway/_index.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+**剔除(Culling)** - 这种方法对于在丢失曲线细节时保留绝对点值非常有用。在消息处理图中，此方法的数据从左侧到达。一旦本地存储中已存放超过*剔除点*的数据，则一些清除逻辑会移除出其他所有样本（或每`N`个样本）。具体数据的例子包括: [千瓦(kW)](https://en.wikipedia.org/wiki/Watt#Kilowatt), [安培(Amperage)](https://en.wikipedia.org/wiki/Amperage), [伏特(Voltage)](https://en.wikipedia.org/wiki/Voltage), etc
+
+**聚合(Aggregate)** – 当曲线的详细形状不如一段时间内的最小值，最大值，平均值和总和值那么重要时，此方法很有用。在消息处理图中，此方法的数据从左侧到达。一旦本地存储存放超过*聚合点*数据时，一些清除逻辑就对存储的值执行聚合。数据的例子包括: [千瓦时(kWh)](https://en.wikipedia.org/wiki/Kilowatt_hour), [日照时间](https://en.wikipedia.org/wiki/insolation), [流](https://en.wikipedia.org/wiki/Flow_measurement), [CPU时间](https://en.wikipedia.org/wiki/CPU_time), [温度](https://en.wikipedia.org/wiki/Temperature), [风速](https://en.wikipedia.org/wiki/Wind_speed)等等.
```
Comment (brettf) [inline content/zh-cn/designs/gateway/_index.md]: ..and finally, please ensure that these Wikipedia URLs are pointing to the Simplified Chinese version on Wikipedia (if that version exists). Thanks.

File: content/zh-cn/designs/telemetry/_index.md [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+#### IoT解决方案中遥测的成本驱动因素有哪些？
+
+通常，物联网解决方案中最常见的成本驱动因素是设备数量、设备采样和报告频率、必要的*感知到洞察力*或*感知到行动*的遥测处理延迟、[设备数据密度] ({{<ref"/glossary/vocabulary#device-data-density">}})以及[遥测存档]({{<ref "/designs/telemetry_archiving" >}})的保留时间。
+
+#### 每个设备是否与其他设备在报告频率上“主动不对齐”？
+当IoT解决方案中的所有设备都配置相同的报告频率时，则会出现一个具有重大影响的常见错误。为了避免隐藏在这个简单行为中的[相长干涉](http://en.wikipedia.org/w/index.php?title=Constructive_interference)，设备应该只在它唤醒并且在一段随机持续时间过后才开始进行报告。这种启动时间的随机性可以产生一个更为平滑的进入解决方案的感测数据流，防止因为设备在不可避免的区域网络中断或其他对解决方案有影响的中断后恢复通信时产生的相长干涉。
+
+#### 当设备无法连接到其默认的IoT解决方案端点时应该做些什么？
+
+**可预期的中断时长** - 当设备在可预期的时长内无法连接到默认的IoT解决方案端点，该设备应该按照配置进行*设备消息排队*。这与确定设备采样和报告频率的差异时的答案可能是一样的。此外，任何能够执行设备消息排队的设备都应该考虑与做为设备[网关]({{<ref"/designs/gateway">}})的设备相同的算法权衡。当本地存储不足以在预期持续时间内存储所有消息并且将影响所感测的数据时，则需要做出权衡。通常需要考虑的算法类别包括**[先进先出(FIFO)](https://en.wikipedia.org/wiki/FIFO_(computing_and_electronics))**、**剔除(Culling)**和**聚合(Aggregate)**
```
Comment (brettf) [inline content/zh-cn/designs/telemetry/_index.md]: Please ensure that these Wikipedia URLs are pointing to the Simplified Chinese version on Wikipedia (if that version exists).

File: content/zh-cn/designs/device_bootstrap/_index.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+* 通过在制造过程中使用[证书](https://en.wikipedia.org/wiki/Public_key_certificate)可以完成设备的配置，并且可以将这些证书预先映射到设备配置文件。这对于具有大量已知设备的解决方案来说是常见的。
+* 设备在与注册机构通信期间报告其型号和序列号，该信息可以预先映射到设备配置文件。
+* 设备使用BLE或其他本地通信形式来接收有关其身份的信息，例如用户资料。通过移动应用程序处理是最常见的设备配置形式。本地通信可以与制造期间安装证书两者相结合，使得用户关联和注册过程能够在单个步骤中完成。
+
+#### 是否使用设备证书?
+尽管在解决方案中尝试使用证书配置每个设备的想法可能令人生畏，但它是迄今为止配置设备最安全的方式。建立相互身份验证对于防止类似[中间人](https://en.wikipedia.org/wiki/Man-in-the-middle_attack) 攻击这样的威胁是非常重要的。引导设备时，证书应始终是设备标识的首选。
+
+#### 注册机构是否需要支持与现有客户授权方案的进行定制化集成?
+如果**是**，则可以使用应用程序编程接口([API](https://en.wikipedia.org/wiki/Application_programming_interface)) 在现有客户授权方案前面实现本设计的注册机构步骤。然后，此API可以在利用客户现有解决方案的同时执行注册机构的相关工作。
```
Comment (brettf) [inline content/zh-cn/designs/device_bootstrap/_index.md]: Please ensure that these Wikipedia URLs are pointing to the Simplified Chinese version on Wikipedia (if that version exists).

File: content/zh-cn/designs/device_bootstrap/_index.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
+如果**是 - 使用令牌/共享凭证**，在这种情况下，非常重要的一点是使用初始令牌或共享凭证来获取注册到解决方案所需要的最小权限。一旦注册机构验证了初始令牌或共享凭证，此设计的其余步骤需要被遵循。
+
+如果**是 - 使用证书**，则可以以安全的方式制造设备，注册机构也可以弱化甚至不需要了。这说起来容易做起来难，因为许多制造过程都是有意与云断开的。但无论如何，由于制造商引入密钥从而完全去掉注册机构，客户体验和整体系统的简化性都会受益。
+
+#### 用户如何绑定设备?
+
+在几乎所有设备预配置的情况下，我们都需要将设备与用户或已有系统的设备配置文件相关联。这涉及从设备收集附加信息以完成设备注册。收集这些附加信息可以通过以下一种或多种方式完成：
+
+* 通过在制造过程中使用[证书](https://en.wikipedia.org/wiki/Public_key_certificate)可以完成设备的配置，并且可以将这些证书预先映射到设备配置文件。这对于具有大量已知设备的解决方案来说是常见的。
```
Comment (brettf) [inline content/zh-cn/designs/device_bootstrap/_index.md]: Please ensure that these Wikipedia URLs are pointing to the Simplified Chinese version on Wikipedia (if that version exists).

File: content/zh-cn/glossary/vocabulary.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+> 大规模生产设备，并将IoT产品推向市场
+
+### 原型(Prototype)
+> 以教学为目的，获取客户管理层批准开始试点阶段
+
+### 规则(Rule)
+> 由规则引擎执行的逻辑和动作的单一集合
+
+### 规则引擎(Rule Engine)
+> 详见维基百科([Business rules engine](https://en.wikipedia.org/wiki/Business_rules_engine))
```
Comment (brettf) [inline content/zh-cn/glossary/vocabulary.md]: Please ensure that these Wikipedia URLs are pointing to the Simplified Chinese version on Wikipedia (if that version exists).

File: content/zh-cn/designs/command/_index.md (hunk 1 above)
Comment (linjungz) [reply content/zh-cn/designs/command/_index.md]: The link is changed to  ([PPTx](/designs/iot-atlas-patterns.pptx)) .

File: content/zh-cn/designs/gateway/_index.md (hunk 2 above)
Comment (linjungz) [reply content/zh-cn/designs/gateway/_index.md]: The link is changed to  /designs/iot-atlas-patterns.pptx

File: content/zh-cn/designs/telemetry/_index.md (hunk 3 above)
Comment (linjungz) [reply content/zh-cn/designs/telemetry/_index.md]: The link is changed to  /designs/iot-atlas-patterns.pptx

File: content/zh-cn/designs/telemetry_archiving/_index.md (hunk 4 above)
Comment (linjungz) [reply content/zh-cn/designs/telemetry_archiving/_index.md]: The link is changed to  /designs/iot-atlas-patterns.pptx

**Comment by linjungz** [comment]
> These files should be removed as they are no longer used.
> content/zh-cn/designs/command/atlas-command.pptx
> content/zh-cn/designs/device_bootstrap/atlas-bootstrap.pptx
> content/zh-cn/designs/telemetry/atlas-telemetry.pptx
> content/zh-cn/designs/telemetry_archiving/atlas-archiving.pptx

Removed.

**Pull request merged**